Extension/cpptools.json

```diff
@@ -1,3 +1,3 @@
-{
-    "intelliSenseEngine_default_percentage": 100
+{

```

Comment:
git diff says this is a whitespace difference, specifically a carriage return (^M). Undo.

Reply:
I will make sure to undo this change. Do we have any tools for Git to automatically ignore carriage return changes? I feel that checking for carriage returns manually with each commit is wasteful.

Reply:
You could try ```git diff -w | git apply --cached --ignore-whitespace```. However, I would do a quick glance at```git diff``` before creating the PR. 

The common things to look out for with this repo is any changes that you did not make or files that you did not add should not be included with your changes.

Reply:
Thanks! I appreciate the advice + tip. You are totally right about the automatically generated changes and I will make sure to be diligent in scrutinizing these diffs pre-PR.